lib/octopus/proxy_config.rb

```diff
@@ -59,7 +59,7 @@ def current_shard=(shard_symbol)
         fail "Nonexistent Shard Name: #{shard_symbol}" if shards[shard_symbol].nil?
       end
 
-      self.current_slave_group = nil if shard_symbol == :master
+      # self.current_slave_group = nil if shard_symbol == :master
```

Comment:
Curious what you ran into that required this to be commented out?

Reply:
Sorry, I did it just for testing on demo env. Sorry, I should test this in a different branch. Revert it back. :)